README.md

```diff
@@ -32,6 +32,16 @@ simply dropped on read.  The maximum message length depends on the particular
 browser you connect with, but in my testing currently it is slightly smaller
 than 1200 bytes.
 
+## Running the example
+
+```
+cargo run --example echo_server -- --data 127.0.0.1:42424 --http 127.0.0.1:42425 --public 127.0.0.1:42424
+```
+
+Following that check out the example http service and see if you need to update it based on your changes above:
+
+https://github.com/kyren/webrtc-unreliable/blob/f848b73c4f40222682596855ae045cfec940d7e4/examples/echo_server.html#L9
```

Comment:
But you should *never* need to update it, if you got to index.html somehow, the server that serves "/index.html" also serves "/new_rtc_session" so it just has to work, right?

Reply:
Also, I think it's more evocative to use ports 12345 for UDP and 8080 for HTTP, to show that the HTTP port is HTTP.  8080 is extremely common as a non-root available HTTP port, and 12345 is more obviously a made up port with no meaning.